src/v1/combo-chart/combo-chart.ts

```diff
@@ -0,0 +1,669 @@
+/**
+ * @file Combo Chart Implementation from chart.js library
+ *
+ * @fileoverview
+ *
+ * @author Rohit Singh <rohit.singh@thoughtspot.com>
+ *
+ * Copyright: ThoughtSpot Inc. 2023
+ */
+
+import {
+  ChartColumn,
+  ChartConfig,
+  ChartModel,
+  ChartToTSEvent,
+  ColumnType,
+  CustomChartContext,
+  DataPointsArray,
+  getChartContext,
+  PointVal,
+  Query,
+  ValidationResponse,
+  VisualProps,
+} from "@thoughtspot/ts-chart-sdk";
+import Chart from "chart.js/auto";
+import { htmlLegendPlugin } from "./legend-plugin";
+import ChartDataLabels from "chartjs-plugin-datalabels";
+import _, { find } from "lodash";
+
+const logger = console;
+
+Chart.register(ChartDataLabels);
+
+let globalChartReference: Chart;
+
+const visualPropKeyMap = ["datalabels", "hideLegend"];
+
+const availableColors = [
+  "#FF0000",
+  "#00FF00",
+  "#0000FF",
+  "#FFFF00",
+  "#FF00FF",
```

Comment:
a visual marker in the form of color name to define color pallette would have been helpful.